test/tests/httptests.ts

```diff
@@ -9,6 +9,12 @@ import * as path from 'path';
 
 let sampleFilePath: string = path.join(__dirname, 'testoutput.txt');
 
+const nodeVersionsWithCertificateErrors = [6, 8];
+if (nodeVersionsWithCertificateErrors.find((nodeVersion) => process.version.startsWith(`v${nodeVersion}.`))) {
+    console.log('Setting NODE_TLS_REJECT_UNAUTHORIZED = 0 to avoid certificate errors on this node version');
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
This changes is highly insecure.

Reply:
Fixed it - the pipeline now uses the HTTP protocol instead of HTTPS in the redirect tests on node 6 / 8. This also fixes the issue.